lib/generators/frontier_model/templates/model.rb

```diff
@@ -4,6 +4,12 @@ class <%= model_configuration.as_constant %> < ActiveRecord::Base
   acts_as_paranoid
 
 <% # The whitespace here is very important, we only want to include a blank line if there are some enums, etc -%>
+<% if model_configuration.attributes.flat_map(&:constants).any? -%>
+<% model_configuration.attributes.flat_map(&:constants).each do |constant| -%>
+  <%= constant.model_implementation %>
+<% end -%>
```

Comment:
I actually still need to get it working. WS is REALLY frustrating in these templates.

What I get from above is

```ruby
ONE = 2
TWO = 1

# other stuff
```

without the WS and if as above I'd get either

```
ONE = 2

TWO = 1

# other stuff
```

or

```
ONE = 2
TWO = 1
# other stuff
```

or if there's no constants without the if you'd get

```
class Whatever


    # other stuff
end
```

see the extra WS at the top of the class.